metrics/config_observability.go

```diff
@@ -77,6 +80,7 @@ func NewObservabilityConfigFromConfigMap(configMap *corev1.ConfigMap) (*Observab
 		cm.AsString("logging.revision-url-template", &oc.LoggingURLTemplate),
 		cm.AsString("logging.request-log-template", &oc.RequestLogTemplate),
 		cm.AsBool("logging.enable-probe-request-log", &oc.EnableProbeRequestLog),
+		cm.AsBool("logging.enable-request-logging", &oc.EnableRequestLogging),
```

Comment:
Will we need to do "smart" defaulting here? If the template is set I guess we should default to `true`, otherwise to `false` to be able to upgrade safely?

Reply:
@markusthoemmes depends on the semantics. In other frameworks eg. log4j you might set LEVEL.OFF to disable output, that does not mean that configuration does not exist. I am not sure if we want to externalize this `http` convention about the template. I might want to keep the template but only turn the logging off. The next time I might want to turn it by applying the existing, no need to re-apply the template.

Reply:
I agree with what you proposed.

Reply:
Yep, that indeed sounds good!